packages/apps/plugins/plugin-debug/src/DebugPlugin.tsx

```diff
@@ -230,6 +231,40 @@ export const DebugPlugin = (): PluginDefinition<DebugPluginProvides> => {
               if (object === 'devtools' && settings.values.devtools) {
                 return <DevtoolsArticle />;
               }
+
+              if (!object || typeof object !== 'object' || !settings.values.debug) {
+                return null;
+              }
+
+              if ('space' in object && isSpace(object.space)) {
+                return (
+                  <DebugSpace
```

Comment:
I think this should probably be combined w/ the `main` one to avoid the `onAddObjects` logic drifting. If you look at some of the other surface providers they do something like `const primary = data.active ?? data.object` prior to checking the types.